identity-controller/src/app/services/email.service.ts

```diff
@@ -0,0 +1,35 @@
+import * as nodemailer from 'nodemailer';
+import Mail = require('nodemailer/lib/mailer');
+
+export class EmailService {
+  transporter: Mail;
+  constructor(opts: {
+    host: string;
+    port: number;
+    user: string;
+    pass: string;
+  }) {
+    const { host, port, user, pass } = opts;
+    const transport = nodemailer.createTransport({
+      host: host,
+      port: port,
+      secure: false,
+      auth: {
+        user: user,
+        pass: pass
+      }
+    });
+    this.transporter = transport;
+  }
+
+  async mailInvite(opts: { address: string; url: string }) {
+    const { address, url } = opts;
+    const mail = await this.transporter.sendMail({
+      from: 'Identity Kit POC <no-reply@gov.bc.ca>',
+      to: address,
+      subject: 'Welcome to the Identity Kit POC test.',
+      html: `<h3>Welcome to identity kit POC test.<h3> <p>Click here to get started <a href="${url}">Link</a></p>`
```

Comment:
How hard would it be to pull this from a resource file?
Not really a request for a change, mostly thinking about the future, when we will need to change the email template...

Reply:
There's an e-mail templating engine called PUG that will allow us to do that. Nodemailer has options for changing the email rendering/templating engine. I didn't want to add the overhead for round one but it is considered in the architecture. Pug template files could potentially be managed by the user in the future. Would be a bit of work though.

Reply:
Sounds good. i think we can leave it as-is for now, but it sounds like we definitely have options for introducing email templates in the future.